Software/src/src.pro

```diff
@@ -104,34 +107,13 @@ win32 {
     LIBS    += -lpsapi
     LIBS    += -lwtsapi32
 
-    CONFIG(msvc) {
-        QMAKE_POST_LINK = cd $(TargetDir) && \
-                cp -f \"../../lib/prismatik-hooks.dll\" ./ && \
-                cp -f \"../../lib/libraryinjector.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Core$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Gui$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5SerialPort$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Widgets$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Network$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icudt54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuin54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuuc54.dll\" ./
-    } else {
-        QMAKE_POST_LINK = cd $(DESTDIR) && \
-                cp -f \"../../lib/prismatik-hooks.dll\" ./ && \
-                cp -f \"../../lib/libraryinjector.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Core$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Gui$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5SerialPort$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Widgets$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/Qt5Network$${DEBUG_EXT}.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icudt54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuin54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/icuuc54.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libwinpthread-1.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libgcc_s_dw2-1.dll\" ./ && \
-                cp -f \"$${QTDIR}/bin/libstdc++-6.dll\" ./
-    }
+    QMAKE_POST_LINK = cd $(TargetDir)$$escape_expand(\r\n)\
+        set VCINSTALLDIR=$(VcInstallDir)$$escape_expand(\r\n)\
+        $$[QT_INSTALL_BINS]/windeployqt --no-angle --no-svg --no-translations \"$(TargetName)$(TargetExt)\"$$escape_expand(\r\n)\
+        del opengl32sw.dll$$escape_expand(\r\n)\
+        rd /s /q bearer $$escape_expand(\r\n)\
+        rd /s /q imageformats $$escape_expand(\r\n)\
```

Comment:
Ahh, makes sense. But yeah, I decided to rewrite those commands to native Windows ones, this way people don't need to have MinGW or any other posix environment installed in their PATH.